pkg/fleet/installer/installer.go

```diff
@@ -130,14 +127,14 @@ func (i *installerImpl) Install(ctx context.Context, url string) error {
 	if err != nil {
 		return fmt.Errorf("could not pre-setup package: %w", err)
 	}
-	err = checkAvailableDiskSpace(mininumDiskSpace, i.packagesDir)
-	if err != nil {
-		return fmt.Errorf("not enough disk space: %w", err)
-	}
 	pkg, err := i.downloader.Download(ctx, url)
 	if err != nil {
 		return fmt.Errorf("could not download package: %w", err)
 	}
+	err = checkAvailableDiskSpace(pkg, i.packagesDir)
```

Comment:
nit: it's a bit tricky that the method called downloader.Download gets only the metadata